how-to/sending/creating-prefilled-transaction-beta.md

```diff
@@ -0,0 +1,47 @@
+---
+title: How To Create a Pre-Filled Transaction Link (beta)
+description: Transactions
+tags:
+  - Beta
+  - Transactions
+priority: 40
+date_published: '2020-08-18'
+date_modified: '2020-08-18'
+---
+
+<Alert>
+
+This feature is currently only found on MyCrypto beta, which you can find at [beta.mycrypto.com](https://beta.mycrypto.com/).
+
+</Alert>
+
+Pre-filled transactions can be created by appending certain parameters into the query section of a URL.
+
+### Currently supported parameters
+
+* `type` - The type of prefilled transaction (use **resubmit** in most cases).
+* `gasPrice` - The *gas price* of the transaction (should be in hex wei format. ex: **0x059682f000**).
+* `gasLimit` - The *gas limit* field of the transaction (should be in hex format. ex: **0xcb56**).
+* `chainId` - The *chainid* of the network the transaction should be conducted on (ex: **1**).
+* `value` - The *value* of the transaction (should be in hex wei format. ex: **0x8AC7230489E80000**).
+* `nonce` - The *nonce* field of the transaction (should be in hex format. ex: **0xD8**).
+* `to` - The *to* address of the transaction.
+* `from` - The *from* address of the transaction (should correspond to one of the accounts you have added to your dashboard).
+* `data` - The *data* field of the transaction (should be in hex format. ex: **0xa9059cbb0000000000000000000000005dd6e754d37bababeb95f34639568812900fec79000000000000000000000000000000000000000000000104f6e0a229913de8a2**).
+
+An example:
+
+ETH Transaction
+<https://beta.mycrypto.com/send?type=resubmit&gasLimit=0x5208&chainId=1&nonce=0xD8&gasPrice=0x059682f000&from=0x5197B5b062288Bbf29008C92B08010a92Dd677CD&to=0x5197B5b062288Bbf29008C92B08010a92Dd677CD&value=0x8AC7230489E80000&data=0x>
+
+ERC20 Transaction
+<https://beta.mycrypto.com/send?type=resubmit&gasLimit=0x5208&chainId=1&nonce=0xD8&gasPrice=0x059682f000&from=0x5197B5b062288Bbf29008C92B08010a92Dd677CD&to=0x6B175474E89094C44Da98b954EedeAC495271d0F&value=0x0&data=0xa9059cbb0000000000000000000000005197B5b062288Bbf29008C92B08010a92Dd677CD000000000000000000000000000000000000000000000104f6e0a229913de8a2>
+
+## Final notes
+
+* We only support the parameters listed above. 
```

Comment:
I feel like you can just remove this section, since it's pretty self-explanatory.